Align touchcancel and pointercancel examples (#30698)
mdn/content: files/en-us/web/api/element/pointercancel_event/index.md

- The device's screen orientation is changed while the pointer is active.
- The browser decides that the user started pointer input accidentally. This can happen if, for example, the hardware supports palm rejection to prevent a hand resting on the display while using a stylus from accidentally triggering events.
- The {{cssxref("touch-action")}} CSS property prevents the input from continuing.

> - When the user interacted with too many simultaneous pointers the browser can fire this event for all existing pointers (even if they are still touching the screen).

Reviewer: ```suggestion
- When the user interacts with too many simultaneous pointers, the browser can fire this event for all existing pointers (even if the user is still touching the screen).
```
